circuit/types/integers/src/abs_checked.rs

```diff
@@ -100,9 +100,11 @@ mod tests {
     }
 
     fn run_test<I: IntegerType + UnwindSafe>(mode: Mode) {
+        let mut rng = TestRng::default();
```

Comment:
Not entirely sure if or where this function is used but if it's called multiple times the rng should be passed in.

Reply:
this one's fine, as it creates an individual test case every time it's called